packages/legend-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -80,9 +80,19 @@ const AssociationPropertyBasicEditor = observer(
   }) => {
     const { association, property, selectProperty, isReadOnly } = props;
     const editorStore = useEditorStore();
+    const checkDuplicateValue = (val: string): boolean => {
```

Comment:
--> `checkDuplicatedValues`

Reply:
I think the value of the function shouldn't be of type `property` because each `property` will have different `uuid` so this function will always return `false`